packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js

```diff
@@ -4828,9 +4828,117 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
   {
     component: "Search input",
     description: "Removed component",
-    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5506 ",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5506",
     details: "The search input component was removed in favor of the text input group component.",
     repo: "HTML/CSS",
     fixedWithCodeMod: false
-  }
+  },
+  {
+    component: "Quickstarts",
+    description: "Made compatible with V5",
+    pullRequestURL: ["https://github.com/patternfly/patternfly-quickstarts/pull/250", "https://github.com/patternfly/patternfly-quickstarts/pull/252"],
+    repo: "Quickstarts",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Quickstarts",
```

Comment:
```suggestion
    component: "Quick starts",
```

Reply:
@jessiehuff is the proper name two words or one? Lol i never know - but since the repo name is quickstarts i tend to assume its one word?

Reply:
It's two words: "Quick starts" :)